code/modules/cargo/packs.dm

```diff
@@ -1133,6 +1143,84 @@
 	crate_name = "sandstone blocks crate"
 
 //////////////////////////////////////////////////////////////////////////////
+////////////////////////////// Munitions /////////////////////////////////////
+//////////////////////////////////////////////////////////////////////////////
+
+/datum/supply_pack/munitions
+	group = "Munitions"
+	crate_name = "munitions crate"
+	sensitivity = 2
+
+/datum/supply_pack/munitions/he
+	name = "MAC Cannon Shell (High Explosive)"
+	cost = 1000
+	contains = list(/obj/structure/shell)
+	access = access_munitions
+	crate_type = /obj/structure/closet/crate/secure
+	sensitivity = 1
```

Comment:
Don't know how I feel about having only HE shells be availible in systems other than NT.